frontend/src/shared/components/tasque-task-editing/task-editing.component.html

```diff
@@ -0,0 +1,153 @@
+<tasque-sidebar name="task-editing" btnText="Edit Task" btnClass="btn stroke">
+  <form class="edit-task-dialog" [formGroup]="editTaskForm">
+    <div class="task-summary-container">
+      <span class="item-label"> PR-{{ taskProject.id }}</span>
```

Comment:
I think it's better to use project.Name here. Users will never interact directly with project's Ids